browser/src/map/Clipboard.js

```diff
@@ -972,6 +932,10 @@ L.Clipboard = L.Class.extend({
 				text = text.substring(idx, text.length);
 			textHtml = text;
 		}
+
+		if (!app.sectionContainer.testing)
```

Comment:
what is that `testing` flag?

Reply:
It's "true" when Cypress tests is active. We normally remove styles from HTML string but in case of testing, we don't. So we can see the affect of the uno commands inside the incoming HTML.